lm_eval/tasks/superglue.py

```diff
@@ -229,6 +246,89 @@ def aggregation(self):
             "acc": acc_all
         }
 
+
+class ReCoRD(HFTask):
+    DATASET_PATH = "super_glue"
+    DATASET_NAME = "record"
+
+    def has_training_docs(self):
+        return True
+
+    def has_validation_docs(self):
+        return True
+
+    def has_test_docs(self):
+        return True
+
+    def training_docs(self):
+        # In ReCoRD, each doc manifests multiple "examples" in the context of few shot example packing.
+        # Each doc consists of multiple answer candidates, each of which is scored yes/no.
+        # Hence, we one "doc" for each (context + passage, answer) pair.
+        # Moreover, we only use the correct answers for context packing
+        # (This is not an issue for evaluation, where we can directly score multiple candidates at once).
+        if self.has_training_docs():
```

Comment:
This has_training_docs call looks a bit redundant but it's not a big deal since it's always true here anyways

Reply:
Not a blocking issue though